app/views/admin/suppliers/new.html.erb

```diff
@@ -0,0 +1,11 @@
+<% content_for :before_content, govuk_back_link(text: 'Back', href: admin_suppliers_path) %>
+
+<%= @value %>
+<%= form_with model: @new_supplier_form, url: admin_new_supplier_path, method: :post do |f| %>
+  <%= f.govuk_text_field(
+        :name,
+        width: 'three-quarters',
```

Comment:
Didn't we agree on double quotes throughout?

Reply:
Ah yes, no linter here to save me